packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -239,27 +232,27 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
   notEnoughReplacementsTitle: {
     message: 'Not enough replacements available',
     context:
-      'Title of modal when a user tries to replace more questions than are available in the pool',
-  },
-  noReplacementsMessage: {
-    message:
-      'There are no more similar questions available to replace the selected questions with.',
-  },
-  notEnoughReplacementsMessage: {
-    message:
-      "You've selected { selected, number } { selected, plural, one { question } other { questions } } to replace, but {available, plural, =0 { don't have questions } one { only have 1 question } other { only have { available } questions } } available to replace them with.",
-    context:
-      'Message of modal when a user tries to replace more questions than are available in the pool',
-  },
-  addMoreQuestionsWithEmptyPool: {
-    message: 'You can add more questions, or delete existing questions.',
-    context: 'Message of modal when a user tries to replace questions but the pool is empty',
-  },
-  addMoreQuestionsWithNonEmptyPool: {
+      'Title of modal when a user tries to replace more questions than are available',
+  },
+  // noReplacementsMessage: {
+  //   message:
+  //     'There are no more similar questions available to replace the selected questions with.',
+  // },
+  // notEnoughReplacementsMessage: {
+  //   message:
+  //     "You've selected { selected, number } { selected, plural, one { question } other { questions } } to replace, but {available, plural, =0 { don't have questions } one { only have 1 question } other { only have { available } questions } } available to replace them with.",
+  //   context:
+  //     'Message of modal when a user tries to replace more questions than are available in the pool',
```

Comment:
after doing manual QA will come back to this and make sure all scenarios are still covered